### DISCUSSION TITLE
22000.258.30.0

### DISCUSSION THREAD

**Discussion by valinet** [valinet/ExplorerPatcher#99] [Announcements]
Tested on build: 22000.258.

* Reworked settings framework
  * More settings are available to customize
  * Most setting changes take effect immediately
* Implemented Windows 10 window switcher (Alt+Tab)
* GUI
  * Revamped GUI, now the interface is split by categories and is displayed on two columns
  * Regular items do not display a "+" sign anymore at the beginning of their label
  * The current choice is ticked in the drop down menu
  * Regions are now calculated correctly
  * Solved memory leaks 
* Option to disable immersive context menus (#96)
* General bug fixes

Please consult the [README](https://github.com/valinet/ExplorerPatcher/blob/master/README.md) for more details.

A detailed change log is available [here](https://github.com/valinet/ExplorerPatcher/blob/master/CHANGELOG.md). 

<hr /><em>This discussion was created from the release <a href='https://github.com/valinet/ExplorerPatcher/releases/tag/22000.258.30'>22000.258.30.0</a>.</em>

**Comment by distilledliquor** [comment]
I don't know why but the version doesn't change even if I installed 22000.258.30.2

**Comment by Gaurav-Original-ClassicShellTester** [comment]
The new UI is just AWESOME and solves all screen height issues and organizes settings neatly. Thank you!

I am going to stop loading VistaSwitcher on Windows 11 at startup and give your Alt+Tab a try as Windows 10's Alt-Tab I found pretty decent (except for lacking power user features of VistaSwitcher, some of which you've already implemented) :))

**Reply by valinet** [reply]
Yeah, thanks, see if it fits in with your demands. I won't implement anything crazy in it, let me preface this, I want to keep it as simple and as true to the original as possible while implementing only the obvious and necessary quality of life improvements on top of that.

**Reply by valinet** [reply]
And yeah, I also think the GUI is now optimal, neatly and logically organized, not taking a lot of space. It was hard to make it right, but since it is not hardcoded, adding stuff to it is fairly easily. Interested folks should check out the https://github.com/valinet/ExplorerPatcher/blob/master/ExplorerPatcher/settings.reg file, that's what holds all the stuff that's drawn by the GUI. The best part is that it is written in `.reg` file syntax, so it is also valid to run it and just restore everything to default.

**Comment by Mervinj82** [comment]
Thanks for the update... New UI looks nice. Well done. 👏👍

**Comment by ParanormalBanana** [comment]
Hey my computer has just done a windows update and Explorer Patcher stopped working. The normal windows 11 taskbar is up, everything is as is dxgi isn't loaded at all.

![image](https://user-images.githubusercontent.com/7958496/137636918-1eb93f1d-55c5-422c-8394-eaf2b0fac76a.png)
![image](https://user-images.githubusercontent.com/7958496/137636930-6e09574a-ed6c-4da2-a86a-88c5a9b7b20d.png)

**Reply by grzesimaj-74** [reply]
In the Releases section, you will find a download for a DLL called dxgi.dll. It has been tested on the latest stable build of Windows 11 (22000.282)?

**Reply by grzesimaj-74** [reply]
Delete In the Releases section, you will find a download for a DLL called dxgi.dll. It has been tested on the latest stable build of Windows 11 (22000.258)

**Comment by kolunio82** [comment]
Nothing after taskkill /f /im explorer.exe, just explorer is closed.

**Reply by sorryjack** [reply]
call again the explorer by typing "explorer"

**Comment by kolunio82** [comment]
Doesn't work. I have dll in windows folder, I tried step by step few times.
my win is 1000.22000.258.0


śr., 20 paź 2021, 05:09 użytkownik Grzegorz Majewski <
***@***.***> napisał:

**Reply by ParanormalBanana** [reply]
Hey the latest build of windows doesn't work, there is an issue with a parameter that is not available anymore, so I think Valinet is working on a fix to get rid of this value altogether. 

For now, the solution is to uninstall KB5006746 update and pause updates until problem is resolved. I did it and it worked perfectly.

**Reply by sorryjack** [reply]
same as mine, notification about downloading resources shown up but no ending notification. Then I do windows update, and the finished downloading notification shown up. then the explorer restart and my taskbar icons are left aligned. I think its working now.

**Comment by Mattitla** [comment]
One reason why I'm using this Explorer Patcher dll is the small icons taskbar. Original taskbar probably is ok for touch-interface, but too big for mouse users(my opinion). 

I did have same problem with latest dll released. Changing options didn't do anything with latest Consumer Win11(non insider). I decided to try StartAllBack and I didn't like it. Uninstalled it and now I have small icons taskbar. How this all works, I don't know.
